cmd/mdatagen/templates/component_test.go.tmpl

```diff
@@ -1,5 +1,9 @@
 // Code generated by mdatagen. DO NOT EDIT.
 
+{{- if len .Status.UnsupportedPlatforms }}
+//go:build !({{range $i, $v := .Status.UnsupportedPlatforms}}{{if $i}} || {{end}}{{.}}{{end}})
```

Comment:
This seems incorrect. It should be logical AND not OR for the unsupported platforms list.
```suggestion
//go:build {{ range $i, $v := .Status.UnsupportedPlatforms }}{{ if $i }} && {{ end }}!{{ . }}{{ end }}
```

Reply:
Oh actually, your solution is good as well since you apply `!(...)` over it, but for some reason `()` are not rendered... Maybe removed by `go fmt`... I still think `!a && !b` is cleaner than `!(a || b)` though

Reply:
Agreed